cg/constants/report.py

```diff
@@ -220,15 +220,10 @@
 ]
 
 
-REQUIRED_SAMPLE_METADATA_TAXPROFILER_FIELDS: list[str] = (
-    _REQUIRED_SAMPLE_METADATA_SEQUENCING_FIELDS
-    + [
-        "average_read_length",
-        "mapped_reads",
-        "mean_length_r2",
-        "million_read_pairs_after_filtering",
-    ]
-)
+REQUIRED_SAMPLE_METADATA_TAXPROFILER_FIELDS: list[str] = [
+    "input_amount",
```

Comment:
I missed that those two values were not there before. Was there a reason for that?

Reply:
They were, they are part of `_REQUIRED_SAMPLE_METADATA_SEQUENCING_FIELDS`, which I removed now since that constant includes a couple of more metrics

Reply:
From now on, only these two metrics will be required to have a value to generate the report

Reply:
Sounds good!